_layouts/post.liquid

```diff
@@ -12,6 +12,14 @@
       <div class="col-md-8">
         <main class="blog">
           <article>
+            <h1>{{ title }}</h1>
+            {% for post in collections.posts %}
+              {% if title == post.data.title %}
+                <em>{{ post.date | dateFormat: "d LLLL yyyy" }}</em>
+              {% endif %}
+            {% endfor %}
```

Comment:
Can we simplify this by taking the date from `page.date`?

```suggestion
            <em>{{ page.date | dateFormat: "d LLLL yyyy" }}</em>
```

Reply:
Updated to page.data for the date in the layout